src/main/java/com/neovisionaries/i18n/CountryCode.java

```diff
@@ -2108,6 +2108,14 @@ public Locale toLocale()
      */
     XI("Northern Ireland", "XXI", -1, Assignment.USER_ASSIGNED),
 
+    /**
+     * <a href="http://en.wikipedia.org/wiki/United_Kingdom">United Kingdom (excluding Northern Ireland)</a>
+     * [<a href="https://ec.europa.eu/taxation_customs/sites/taxation/files/use_of_gb_and_xi_codes_guidance.pdf">XU</a>, XXU, -1,
+     * User assigned]
+     *
+     */
+    XU("United Kingdom (excluding Northern Ireland)", null, -1, Assignment.USER_ASSIGNED),
```

Comment:
should the null be XXU (as per your own javadoc above)?

Reply:
Good catch @pjfanning - have just pushed a fix for this.